pkg/cvo/sync_worker_test.go

```diff
@@ -189,3 +189,74 @@ func Test_runThrottledStatusNotifier(t *testing.T) {
 	case <-time.After(100 * time.Millisecond):
 	}
 }
+
+func Test_equalDigest(t *testing.T) {
```

Comment:
Since your tests are so complete already, how about empty repo A and B:
>{
>	name:      "A empty repo",
>	pullspecA: "@sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
>	pullspecB: "example.com:@sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
	expected:  false,
>},

Reply:
I've added two of those with f22f5725 -> f5addd09.  But I expect them to match, because they have the same digest.

Reply:
And pushed f5addd09 -> b047d427 to get [that expected sense][1] on the two new cases.

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-version-operator/490/pull-ci-openshift-cluster-version-operator-master-unit/1338886325352271872#1:build-log.txt%3A175